services/registration/src/models/branch.ts

```diff
@@ -20,17 +21,43 @@ export interface Branch {
 }
 
 const branchSchema = new Schema<Branch>({
-  name: { type: String, required: true },
-  type: { type: String, required: true, enum: BranchType },
+  name: {
+    type: String,
+    required: true,
+  },
+  hexathon: {
+    type: Number,
+    required: true,
+  },
```

Comment:
Here, the type should be `type: Schema.Types.ObjectId` to specify that the type connects to a different collection